### PR TITLE
Added File Does Not Exist error in troubleshooting

### PR DESCRIPTION
**Pull request by rjusselrelic**
Because I cannot read instructions I spent enough time on this error to precipitate this PR.   In reviewing my changes to this page I noticed the Technical Detail: that explains my issue, and makes me feel very silly. Im submitting this proposed change anyway.

## Summary

[NOTE]: # ( Provide a brief overview of the changes. )

## Checklist
[NOTE]: # ( Just check the ones applicable to this pull request. )
- [x] Documentation updated
- [ ] AWS Deployment configuration updated
- [ ] GCP Deployment configuration updated
- [ ] Azure Deployment configuration updated

## Related Issues
[NOTE]: # ( Provide links to any related Github issues. )

## Related Pull Requests
[NOTE]: # ( Provide links to any related pull requests. )

## Reviewers
[NOTE]: # ( Mention reviewers here! )

### CODE REVIEW

File: GETTING_STARTED.md [hunk 1]
```diff
@@ -105,6 +105,19 @@ See ‘docker run --help’.
...
+```console
+$ docker run -it\
+    -v $HOME/demo-deployer/configs/:/mnt/deployer/configs/\
+    --entrypoint ruby ghcr.io/newrelic/deployer main.rb -d <demo-url>
+(...)
+[ERROR] Infrastructure validation has failed
+File '/path/to/jsmith.pem' doesn't exist
+The pem key file '/path/to/jsmith.pem' does not have the correct permissions.  To fix run 'chmod 400 /path/to/jsmith.pem''.
+```
+This may indicate the incorrect path in your configuration. NOTE: all paths in the config should reference the mounted volume file structure in the above examle, the key paths would be set as `/mnt/deployer/configs/jsmithUsWest2.pem`.
```
Comment (polfliet) [inline GETTING_STARTED.md:118]: typo: examle -> example

**Review by polfliet — COMMENTED** [review]
Thanks for your contribution @rjusselrelic! 
I just noticed a small typo but it would be good to add this to the troubleshooting section, thanks

File: GETTING_STARTED.md (hunk 1 above)
Comment (rjusselrelic) [reply GETTING_STARTED.md]: Great catch, thank you!  I've updated and re-submitted.

**Review by rjusselrelic — COMMENTED** [review]
Re-submitting with fixed typo

**Review by polfliet — APPROVED** [review]

**Comment by polfliet** [comment]
Thanks for your contribution @rjusselrelic !

**Pull request merged**